### DISCUSSION TITLE
Chocolatey choco completion

### DISCUSSION THREAD

**Discussion by Cree2K** [chrisant996/clink#537] [Ideas]
I use choco for installing and upgrading programmes.
This will be a great to have command line completion.
https://chocolatey.org

**Comment by chrisant996** [comment]
Clink itself doesn't include completions for other programs.
See [Popular Scripts](https://chrisant996.github.io/clink/clink.html#popular-scripts) for more info.
You're looking for [clink-completions](https://github.com/vladimir-kotikov/clink-completions) which provides completions for chocolatey and many other things.
